src/map/status.c

```diff
@@ -7813,6 +7813,15 @@ static int status_change_start_sub(struct block_list *src, struct block_list *bl
 				val3 = total_tick/1000; //Tick duration
 				tick_time = 1000; // [GodLesZ] tick time
 				break;
+
+#ifdef RENEWAL
+			case SC_POEMBRAGI:
+				// val1 = skill lv
+				val2 = val1 * 2; // Cast time reduction
```

Comment:
I have been wondering whether we could/should/want to move these vals to `SkillData2` and `SkillData3` instead of defining them here. (For every skill, not only bragi)